### ISSUE TITLE
Create raw list Schema

### ISSUE THREAD

**Issue by l00p1n6** [marshmallow-code/marshmallow#2212]
How can one create Schema derived class from a list only data?

For input with a field value, i.e. dictionary,  e.g.:
```
{
  "string_list": [
    "one",
    "two",
    "three"
  ]
}
```
one can use;
```
class StringListSchema(Schema):
    string_list = fields.List(fields.Str())
```

But how can one so it when input is just a list:
```
[
  "one",
  "two",
  "three"
]
```
?

NB: both are valid JSONs

**Comment by lafrech** [comment]
I don't think you can do that with marshmallow. I'm pretty sure there was once a discussion about it, maybe involving sort of merging Schema and Field. I can't find it.

You could probably use a `pre_load` to wrap the payload into a dict with a single key and using that key as name for the List field in the schema, but this sucks, of course.

**Comment by sloria** [comment]
for deserialization, you could call `deserialize` on the `List` field.

```python
from marshmallow import fields


field = fields.List(fields.String())

field.deserialize(["one", "two", "three"])  # ['one', 'two', 'three']
```

**Comment by sloria** [comment]
note: this is the proposed in https://github.com/marshmallow-code/marshmallow/issues/1694
